CadRevealFbxProvider.Tests/FbxProviderTests.cs

```diff
@@ -7,33 +7,33 @@ namespace CadRevealFbxProvider.Tests;
 using CadRevealComposer.ModelFormatProvider;
 using CadRevealComposer.Operations;
 using CadRevealComposer.Primitives;
-using CadRevealFbxProvider.Attributes;
+using Attributes;
 using System.Numerics;
 
 [TestFixture]
 public class FbxProviderTests
 {
-    private static readonly DirectoryInfo outputDirectoryCorrect = new DirectoryInfo(@".\TestSamples\correct");
-    private static readonly DirectoryInfo inputDirectoryCorrect = new DirectoryInfo(@".\TestSamples\correct");
+    private static readonly DirectoryInfo OutputDirectoryCorrect = new("TestSamples//correct");
```

Comment:
Do we need double slashes here? Is there a diff between singel and double forward slash?